pom.xml

```diff
@@ -329,6 +329,12 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>org.zeppelin-project.zeppelin</groupId>
```

Comment:
hmm, this is kinda strange

Reply:
I got the idea from https://github.com/apache/spark/blob/cac9b1dea1bb44fa42abf77829c05bf93f70cf20/pom.xml#L2437-L2489 should I revet this and open up a new JIRA to track shade-plugin changes?

Reply:
We should revert this, this PR should only about bumping up thrift version

Reply:
Sure I've reverted that change and create a new JIRA(https://issues.apache.org/jira/browse/ZEPPELIN-3498) to track the same if required.